### PR TITLE
photo id card upload

### PR DESCRIPTION
**Pull request by lidio601**
Relies on https://github.com/lamassu/lamassu-server/pull/151/files

### CODE REVIEW

**Comment by lidio601** [comment]
[Upload ID image to l-s](https://trello.com/c/aiRzeg17/567-upload-id-image-to-l-s)

File: lib/scanner.js [hunk 1]
```diff
@@ -30,13 +33,15 @@ function setConfig (width, height, cam) {
...
-  manatee.register('pdf417', licenses.pdf417.name, licenses.pdf417.key)
+  var licenses = _.get('scanner.manatee.license', configuration)
+  manatee.register('qr', _.get('qr.name', licenses), _.get('qr.key', licenses))
+  manatee.register('pdf417', _.get('pdf417.name', licenses), _.get('pdf417.key', licenses))
+  camera = require('./camera').config(_configuration)
 }
 
 function cancel () {
   cancelFlag = true
+  camera.closeCamera()
```
Comment (joshmh) [inline lib/scanner.js]: Need to check if ``camera`` is defined.

Comment (joshmh) [reply lib/scanner.js]: Once we fix this we can merge into the new branch for testing.

Comment (lidio601) [reply lib/scanner.js]: done

**Comment by lidio601** [comment]
all done

**Pull request merged**